system/genext2fs/genext2fs.SlackBuild

```diff
@@ -70,7 +70,7 @@ rm -rf $PKG
 mkdir -p $TMP $PKG $OUTPUT
 cd $TMP
 rm -rf $PRGNAM-$VERSION
-tar xvf $CWD/$PRGNAM-$VERSION.tar.gz
+tar xvf $CWD/$PRGNAM-$VERSION.tar.gz || tar xvf $CWD/v$VERSION.tar.gz
```

Comment:
Once you follow the github guide, you don't need to provide  || tar xvf $CWD/v$VERSION.tar.gz
anymore, since it will result in consistent tarball

Reply:
fixed, thx